dist/react-slick.js

```diff
@@ -1106,7 +1102,7 @@ return /******/ (function(modules) { // webpackBootstrap
 	    }
 
 	    if (this.props.beforeChange) {
-	      this.props.beforeChange(currentSlide);
+	      this.props.beforeChange(this.state.currentSlide, currentSlide);
```

Comment:
That's really weird. I'm not sure why this change is visible here. I see that that's already merged in master https://github.com/akiran/react-slick/blob/master/dist/react-slick.js#L1104-L1106 

Maybe that comes because I had to merge with the master in this repo and sync my fork.